pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -165,6 +166,8 @@ def restore_training_state(self) -> None:
 
         self.restore_optimizers_and_schedulers()
 
+        self.restore_loops()
```

Comment:
@awaelchli You defined just above the `restore_progress` function which includes "loops" in the comment. Should we use that one?

Reply:
I merged the two functions just now, have a look

Reply:
`progress` is legacy one for now. We should refactor the restoration when we get rid or not of global_step.